storage/Utils/ColumnConfigFile.cc

```diff
@@ -65,8 +71,10 @@ string ColumnConfigFile::Entry::format()
 }
 
 
-bool ColumnConfigFile::Entry::parse( const string & line )
+bool ColumnConfigFile::Entry::parse( const string & line, int line_no )
 {
+    (void) line_no;
```

Comment:
This can also be done with ```int __attribute__((__unused__)) line_no``` in the signature. We might define a macro for that and then enable the compiler warning.